packages/examples/cvat/exchange-oracle/.pre-commit-config.yaml

```diff
@@ -1,11 +1,17 @@
 repos:
-  -   repo: https://github.com/psf/black
-      rev: 22.6.0
+  -   repo: local
       hooks:
-        - id: black
-          language_version: python3.11
-  -   repo: https://github.com/PyCQA/isort
-      rev: 5.12.0
-      hooks:
-        - id: isort
-          language_version: python3.11
\ No newline at end of file
+        - id: lint
+          name: lint
+          entry: ruff check --fix --unsafe-fixes --show-fixes
```

Comment:
Why unsafe fixes are enabled?

Reply:
Here's my reasoning behind enabling unsafe fixes:

I find them quite useful and relatively safe and don't see any reason for disabling the.

Only one change made via `unsafe-fixes` actually made unintended changes which I had to fix by marking this rule as `unfixable` globally.

We can always disable it if it will cause more troubles than use, but unless we'll try, we'll never know, but will be guaranteed to miss out on useful fixes.

Reply:
I can see a problem in this, because if these fixes are applied, e.g. on commit automatically, or you're just applying it blindly, you can silently end up with a broken code. So, you can't "just format" the code, you must check all the changes.

Reply:
There was a confusion on whether `--unsafe-fixes` are committed automatically. After a discussion on a call, we decided to leave them on since commit is prevented in case of any changes present.